config.toml

```diff
@@ -54,6 +54,8 @@ rdi_redis_gears_current_semantic_version = "1.2.6"
 rdi_redis_gears_current_version = "10206"
 rdi_debezium_server_version = "2.1.1.Final"
 rdi_db_types = "cassandra|mysql|oracle|postgresql|sqlserver"
+rdi_cli_latest = "0.107.0"
```

Comment:
You don't use it, only the `rdi_cli_preview`, so is it needed?

Reply:
I want to be able to move easy into master when we release